content/hi/continuous_delivery.md

```diff
@@ -0,0 +1,16 @@
+---
+title: निरंतर डिलीवरी (CD)
+status: Completed
+category: concept
+---
```

Comment:
```suggestion
---
title: निरंतर डिलीवरी (CD)
status: Completed
category: अवधारणा
---
```

Reply:
we can safely localize the category label. Refer [spreadsheet](https://docs.google.com/spreadsheets/d/1MkZXrXre7dFVFmrRIHgHhNOYFEYcJkRaIWc3Dp8M-TQ/edit#gid=0)

Reply:
Sure 👍